universe/Fleet.cpp

```diff
@@ -150,7 +150,8 @@ void Fleet::Copy(std::shared_ptr<const UniverseObject> copied_object, int empire
 
                 const std::list<int>& copied_fleet_route = copied_fleet->m_travel_route;
 
-                if (m_travel_route.empty() && !copied_fleet->m_travel_route.empty())
+                m_travel_route.clear();
+                if (!copied_fleet->m_travel_route.empty())
                     m_travel_route.push_back(moving_to);
```

Comment:
I changed the references in the title and commit from `Clone()` to `Copy()`.

Yes it does something. `Copy()` works on an existing object, so `m_travel_route` may not be empty, particularly when called from [here](https://github.com/freeorion/freeorion/blob/master/universe/Universe.cpp#L2397).  `ShortenRouteToEndAtSystem()` uses `m_travel_route` to calculate `travel_route`.

I could refactor it to remove the use of `ShortenRouteToEndAtSystem()`, but this is a simpler change that just fixes the bug, the day before the release candidate is made.